PublicGitArchive/pga/pga/pga.go

```diff
@@ -23,12 +24,41 @@ type Filter func(Repository) bool
 type Dataset interface {
 	Name() string
 	ReadHeader(r *csv.Reader) error
```

Comment:
```suggestion
	ReadHeader(columnNames []string) error
```

This makes sense after you've added `pga.ForEachRepository`.

Reply:
:+1: